flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -140,6 +140,25 @@
                                                     + "or force materialization(FORCE).")
                                     .build());
 
+    @Documentation.TableOption(execMode = Documentation.ExecMode.STREAMING)
+    public static final ConfigOption<SinkShuffleByPk> TABLE_EXEC_SINK_SHUFFLE_BY_PK =
+            key("table.exec.sink.pk-shuffle")
```

Comment:
`table.exec.sink.key-by-mode`?

Reply:
I've considered several candidates but no most satisfied one.  'key-by-mode' is little subtle, what about 'key-by-pk' ?

Reply:
I prefer 'keyed-shuffle',  what do you think?